src/main/java/frc/robot/RobotContainer.java

```diff
@@ -21,77 +22,113 @@
+ * Should actually be handled in the {@link Robot} periodic methods 
+ * (other than the scheduler calls). Instead, the structure of the 
+ * Robot (including subsystems, commands, and button mappings) should
+ * Be declared here.
  */
 public class RobotContainer {
 
   private static RobotContainer m_robotContainer = new RobotContainer();
 
   // BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=DECLARATIONS
-// The robot's subsystems
-    public final Claw m_claw = new Claw();
-    public final Drivetrain m_drivetrain = new Drivetrain();
+  // The robot's subsystems
+  public final Claw m_claw = new Claw();
+  public final Drivetrain m_drivetrain = new Drivetrain();
+  public final MagicArm m_arm = new MagicArm();
+  public final Camera m_camera = new Camera();
 
-// Joysticks
-private final Joystick rightJoystick = new Joystick(1);
-private final Joystick leftJoystick = new Joystick(0);
+  // Joysticks
+  private final XboxController armController = new XboxController(2);
+  private final Joystick rightJoystick = new Joystick(1);
+  private final Joystick leftJoystick = new Joystick(0);
 
-    // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=DECLARATIONS
+  // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=DECLARATIONS
 
   // A chooser for autonomous commands
   SendableChooser<Command> m_chooser = new SendableChooser<>();
-
-  /**
-   * The container for the robot. Contains subsystems, OI devices, and commands.
-   */
+  SendableChooser<String> m_pathSendableChooser = new SendableChooser<>();
+   // The container for the robot. Contains subsystems, OI devices, and commands.
   private RobotContainer() {
     // BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=SMARTDASHBOARD
     // Smartdashboard Subsystems
-
-
     // SmartDashboard Buttons
-    SmartDashboard.putData("Autonomous Command", new AutonomousCommand());
-    SmartDashboard.putData("ShiftLow", new ShiftLow( m_drivetrain ));
-    SmartDashboard.putData("ShiftHigh", new ShiftHigh( m_drivetrain ));
-    SmartDashboard.putData("OpenClaw", new OpenClaw( m_claw ));
-    SmartDashboard.putData("CloseClaw", new CloseClaw( m_claw ));
-    SmartDashboard.putData("ReverseDrive", new ReverseDrive( m_drivetrain ));
-    SmartDashboard.putData("HalfSpeed", new HalfSpeed( m_drivetrain ));
-    SmartDashboard.putData("QuarterSpeed", new QuarterSpeed( m_drivetrain ));
+    // SmartDashboard.putData("Autonomous Command", new AutonomousCommand());
+    // SmartDashboard.putData("ShiftLow", new ShiftLow(m_drivetrain));
+    // SmartDashboard.putData("ShiftHigh", new ShiftHigh(m_drivetrain));
+    // SmartDashboard.putData("OpenClaw", new OpenClaw(m_claw));
+    // SmartDashboard.putData("CloseClaw", new CloseClaw(m_claw));
+    // SmartDashboard.putData("ReverseDrive", new ReverseDrive(m_drivetrain));
+    // SmartDashboard.putData("HalfSpeed", new HalfSpeed(m_drivetrain));
+    // SmartDashboard.putData("QuarterSpeed", new QuarterSpeed(m_drivetrain));
```

Comment:
These dashboard values are commented in autogenerated code. The next time robotbuilder is ran, the comments will be removed. If these aren't needed anymore, remove them from robotbuilder, run robotbuilder and format all the files please.